backpack-common/src/main/java/net/skyscanner/backpack/calendar2/CalendarParams.kt

```diff
@@ -80,12 +80,28 @@ data class CalendarParams(
         /**
          * Only a single, non-disabled date can be selected.
          */
-        data object Single : SelectionMode
+
+        /**
+         * Accessibility labels are NOT supported in the view version
+         * */
```

Comment:
Just */ – see the other multiline comments

Reply:
Thanks, fixed!